web/packages/operations/src/alarm.ts

```diff
@@ -112,9 +112,8 @@ export const sendMetrics = async (metrics: status.AllMetrics) => {
         metricData.push({
             MetricName: AlarmReason.ToEthereumChannelStale.toString(),
             Value: Number(
-                channel.toEthereum.outbound < channel.toEthereum.inbound ||
-                    (channel.toEthereum.outbound > channel.toEthereum.inbound &&
-                        channel.toEthereum.inbound <= channel.toEthereum.previousInbound)
+                channel.toEthereum.outbound < channel.toEthereum.inbound &&
+                    channel.toEthereum.inbound <= channel.toEthereum.previousInbound
```

Comment:
> when inbound > outbound

This is actually impossible means the bridge has been attacked.

Reply:
I see. Your alarm looks correct, then. Do you know if `previousInbound` works correctly? I'm unsure why the alarm triggered, because the `previousInbound` would have changed.

Reply:
Maybe. In recent commits I change the hardcoded `BlockLatencyThreshold` with value loaded from some env, but it could cause issue with env value set improperly.

I'll keep watch on it.